src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
+    .reduce((prev, style) => {
+      const [key, value] = style.split(':');
+
+      if (key && value) {
+        prev[key.trim()] = value.trim();
+      }
+
+      return prev;
+    }, {});
+
+  return obj;
```

Comment:
The variable name 'obj' is too generic and does not describe its contents. Consider using a more descriptive name like 'styleObject' or 'cssProperties'.